R/merge_data.R

```diff
@@ -58,34 +58,26 @@ merge_data <- function(manifest, treatments, data) {
     identifiers[["template"]]
   )
   for (m_col in duplicated_col) {
-    df_metadata[, m_col] <-
-      df_metadata[, paste0(m_col, ".y")] # parse template values
-    missing_idx <-
-      is.na(df_metadata[, m_col]) | df_metadata[, m_col] %in% c("", "-")
-
-    # add manifest values when missing in template
-    df_metadata[missing_idx, m_col] <-
-      df_metadata[missing_idx, paste0(m_col, ".x")]
-
-    # check for conflicts
-    double_idx <- !(is.na(df_metadata[, paste0(m_col, ".x")]) |
-                      df_metadata[, paste0(m_col, ".x")] %in% c("", "-")) &
-      !(is.na(df_metadata[, paste0(m_col, ".y")]) |
-          df_metadata[, paste0(m_col, ".y")] %in% c("", "-"))
-    if (any(double_idx) && 
-        any(
-          df_metadata[, paste0(m_col, ".x")] != 
-          df_metadata[, paste0(m_col, ".y")], 
-          na.rm = TRUE
-    )) {
-      futile.logger::flog.warn(
+    alt_col <- setdiff(grep(m_col, names(df_metadata), value = TRUE), m_col)
+    df_metadata[[m_col]] <-
```

Comment:
consider adding `missing_idx_a` and `missing_idx_m` to make the code more comprehensive

Reply:
done in [44e1419](https://github.com/gdrplatform/gDRcore/pull/108/commits/44e1419d999db1601da3dabc3d05adafaf175b04)